tests/neg/implicitMatch-syntax.scala

```diff
@@ -0,0 +1,26 @@
+object Test {
```

Comment:
I would add neg tests for implicit matches that are not top level and for implicit matches defined in non `transparent ` methods